### PR TITLE
Remove support for Python 2.7 and test 3.6, 3.7 and 3.8

### PR DESCRIPTION
**Pull request by d0ugal**

### CODE REVIEW

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -78,7 +78,7 @@ Testing
 Testing pyLXD is in 3 parts:
 
 1. Conformance with `PEP 8`_, using the ``tox -e pep8`` command.
-2. Unit tests using ``tox -e py27`` and ``tox -e py3``.
+2. Unit tests using ``tox -e py3``.
```
Comment (albertodonato) [inline doc/source/contributing.rst:81]: we could actually `s/py3/py/` as that also matches python3 by default

File: doc/source/contributing.rst [hunk 2]
```diff
@@ -126,7 +126,7 @@ Some hints on how to run the integration tests:
 1. On Ubuntu it's probably easiest to use the `Multipass`_ snap.
 2. Launch an LTS instance using ``multipass launch -n foo``
 3. Shell into the instance: ``multipass exec foo -- bash``
-4. Install tox and python2.7: ``sudo apt install python-tox python-2.7``
+4. Install tox and python3.6+: ``sudo apt install python-tox python3-dev``
```
Comment (albertodonato) [inline doc/source/contributing.rst:129]: I think the package is just `tox` now, but not sure from which ubuntu release.

File: setup.cfg [hunk 3]
```diff
@@ -15,11 +15,10 @@ classifier =
     Operating System :: POSIX :: Linux
     Programming Language :: Python
     Programming Language :: Python :: 2
```
Comment (albertodonato) [inline setup.cfg:17]: This should also go away?

File: requirements.txt [hunk 4]
```diff
@@ -6,4 +6,3 @@ requests>=2.20.0
 requests-unixsocket>=0.1.5
 requests-toolbelt>=0.8.0
```
Comment (albertodonato) [inline requirements.txt:7]: can we also drop dependency on `six` or is it still used somewhere?

**Review by albertodonato — CHANGES_REQUESTED** [review]

File: doc/source/contributing.rst (hunk 2 above)
Comment (d0ugal) [reply doc/source/contributing.rst:129]: I'll just include the instruction and leave folks to find the install docs if they need it.

**Pull request merged**